e2e-tests/trade-finance-manager/cypress/integration/journeys/deals/deals-search.spec.js

```diff
@@ -282,26 +277,6 @@ context('User can view and filter multiple deals', () => {
     });
   });
 
-  it('search/filter by date received in DD-MM-YYYY format', () => {
```

Comment:
this block was duplicated

Reply:
it's actually not - you can search by 2 different date formats:
- DD-MM-YYYY
- DD/MM/YYYY

it's what the duplication is for, can we revert?

Reply:
I'm confused. the text says it should search for different date formats. but the content inside that test is exactly the same as the one above it

Reply:
I see, you're right! That's my fault. There is a unit test for this on the API level anyway so maybe overkill to have both date formats in E2E test ?

Reply:
I've updated this to include the test with the date being formatted correctly